spec.md

```diff
@@ -199,6 +201,7 @@ Pushing an artifact typically works in the opposite order as a pull: the blobs m
 A useful diagram is provided [here](https://github.com/google/go-containerregistry/tree/d7f8d06c87ed209507dd5f2d723267fe35b38a9f/pkg/v1/remote#anatomy-of-an-image-upload).
 
 A registry MAY reject a manifest of any type uploaded to the manifest endpoint if it references manifests or blobs that do not exist in the registry.
+A registry SHOULD accept a manifest with a `refers` field that references a manifest that does not exist.
```

Comment:
@AkihiroSuda I recall seeing a comment from you that this name is confusing. If you have a suggestion, we'd be happy to use it :)

Reply:
I think the word(s) are fine.. Issue is more related to refers and referrers are not defined as terms in the document and because http referer is so similar but not related .. the definitions of these two new terms should be contrasted with the http use.

Reply:
They've now been defined in the Definitions section.